pytripgui/controller/main_cont.py

```diff
@@ -139,8 +139,8 @@ def open_dicom(self, ddir):
 
         # TODO: RTplan data
 
-        # add cube to the treeview<s
-        self.tree.update_tree()
+        # add cube to the treeviews
+        # self.tree.update_tree()
```

Comment:
Add a comment explaining why the line was commented out.

Reply:
ok